sdcm/nemesis.py

```diff
@@ -1781,6 +1781,14 @@ def is_streaming_started():
         if random.randint(0, 1) == 0:
             self.log.debug('Interrupt the task by pkill')
             self.target_node.remoter.run('sudo pkill -9 -f %s' % task, ignore_status=True)
+            self.log.debug("Stop background repair task by storage_service/force_terminate_repair API")
+            with DbEventsFilter(type='DATABASE_ERROR', line="repair's stream failed: streaming::stream_exception",
```

Comment:
we might start think, of turn this repair expected errors into a group to reuse, somehow.
cause this list is keep changing

Reply:
we need to put all grouped filters in one place, like on file. 
Created a task for that https://trello.com/c/V3Ca21uD/1893-group-common-events-filters